tokens/src/components/ProductTour.json

```diff
@@ -14,29 +14,39 @@
             "amount": 0.3
           }]
         }
-      },
-      "width": {
-        "border": { "value": "8px", "source": "$checkpoint-border-width" },
-        "arrow": { "value": "15px", "source": "$checkpoint-arrow-width" },
-        "max": { "value": "480px", "source": "$checkpoint-max-width" }
-      },
-      "arrow": {
-        "color": {
+      }
+    }
+  },
+  "size": {
+    "product-tour": {
+      "checkpoint": {
+        "width": {
+          "border": { "value": "8px", "source": "$checkpoint-border-width" },
+          "arrow": { "value": "15px", "source": "$checkpoint-arrow-width" },
+          "max": { "value": "480px", "source": "$checkpoint-max-width" }
+        },
+        "arrow": {
           "top": {
             "value": "solid {product-tour.checkpoint.width.arrow.value} {product-tour.checkpoint.width.border.value}",
```

Comment:
Something is wrong with these styles, I assume they are for border? but there is no `border` key. And we should probably separate color and width into separate variables and remove whole border declaration from `size` category

Reply:
done